### PR TITLE
chore: make radio button text more descriptive 

### PR DESCRIPTION
**Pull request by jonasbrunvoll**
Close https://github.com/AtB-AS/kundevendt/issues/19625#issue-2704445305

### Background
Currently in the contact form, `App` or `Travelcard` are the only available options for when asking the user to specify where his ticket is stored.  `App` is too vague and can lead tho misunderstandings. A better solution would be to display the name of the app, or a list of names if the ticket can be stored on several apps. 

#### Illustrations
<details>
<summary>screenshots</summary>

Skjermbilde 2024-11-29 kl  10 53 06

</details>

### Proposed solution
Change  the radio button text from `App` to the app name(s)  where the ticket can be found.

### CODE REVIEW

File: src/translations/pages/contact.ts [hunk 1]
```diff
@@ -52,7 +53,7 @@ export const Contact = {
         ),
 
         app: {
-          title: _('App', 'App', 'App'),
+          title: _('Mobilapp', 'Mobile App', 'Mobilapp'),
```
Comment (adriansberg) [inline src/translations/pages/contact.ts:56]: Should the 'a' in "Mobile app" be capitalized?

File: src/translations/pages/contact.ts [hunk 2]
```diff
@@ -1433,3 +1434,21 @@ export const Contact = {
...
+
+export const Contact = orgSpecificTranslations(ContactInternal, {
+  fram: {
+    ticketControl: {
+      feeComplaint: {
+        ticketStorage: {
+          app: {
+            title: _(
+              'Mobilapp (f.eks. FRAM, Entur)',
+              'Mobile App (e.g., FRAM, Entur)',
```
Comment (adriansberg) [inline src/translations/pages/contact.ts:1446]: Should there be a comma before "FRAM"?

File: src/translations/pages/contact.ts (hunk 1 above)
Comment (jonasbrunvoll) [reply src/translations/pages/contact.ts:56]: No, it should not. Will fix now👍

File: src/translations/pages/contact.ts (hunk 2 above)
Comment (jonasbrunvoll) [reply src/translations/pages/contact.ts:1446]: No, it should not. Will update now👍

**Review by adriansberg — APPROVED** [review]

**Pull request merged**